tensorflow_hub/compressed_module_resolver_test.py

```diff
@@ -17,12 +17,14 @@
 import os
```

Comment:
Please remove .DS_Store from the PR (I don't know how to comment on that file directly ...)